### ISSUE TITLE
55530825 Naruto: The Broken Bond

### ISSUE THREAD

**Issue by SakataGintokiYT** [xenia-project/game-compatibility#416]
Xbox Marketplace - http://marketplace.xbox.com/en-US/Product/Naruto-The-Broken-Bond/66acd000-77fe-1000-9115-d80255530825?nosplash=1
##### Xenia Git (2016/02/18)
http://www.emucr.com/2016/02/xenia-git-20160218.html

##### Log : http://www20.zippyshare.com/v/Xu3iR1Ze/file.html
<img src="http://www.fotoszok.pl/upload/28a84100.jpg" alt="2015-12-13 20_52_51-xenia" style="max-width:50%;">
<img src="http://www.fotoszok.pl/upload/73ccca37.jpg" alt="2015-12-13 20_52_51-xenia" style="max-width:50%;">
<img src="http://www.fotoszok.pl/upload/8aa88d8d.jpg" alt="2015-12-13 20_52_51-xenia" style="max-width:50%;">
<img src="http://www.fotoszok.pl/upload/fd953f9d.jpg" alt="2015-12-13 20_52_51-xenia" style="max-width:50%;">

**Comment by SakataGintokiYT** [comment]
◆Build: 8784daf6
◆Trace Frame / Vulkan / XTR : --> **[55530825_25583.xtr ](https://mega.nz/#!MMpmVYYR!gW6y1Ze7O0R2Ot0DQ2m9Ilu0upzV9_y5l5hsuJ-oEkc)**
◆Log: **[xenia.log](https://mega.nz/#!MB5igS7A!MNKaGwRPGVtvUR7k6bqtyr4O57WyuyDkF27jsQS4dls)**
![bez tytulu](https://user-images.githubusercontent.com/13681488/34365887-87cf2c2c-ea95-11e7-9bd8-b7299c854a54.png)

**Comment by Flam3z** [comment]
Tested On 1.0.2509-master
Just tried running the game and got through the main menu with a very few sounds stuttering but the game as a whole crashed about 10 seconds into the first intro.

**Comment by RandomAnimeGamer** [comment]
Tested on master@9fa41c27b (July 26, 2022) and I'm noticing a crash when I receive the tutorial to press Y for a vertical attack.

**Comment by zerr10** [comment]
![obraz](https://user-images.githubusercontent.com/18278728/215618072-f0503549-f100-4eef-afb5-2734c23c170c.png)

**Comment by zerr10** [comment]
problem vertex ?

space i7 6700 3.4 Ghz RAM 8GB DDR4 2133Mhz Dual Channel Card : GTX 1070 8GB

another 3 photos

1 is when I turn my character towards the village, it redirects the image correctly, but as soon as I turn the camera towards the character, the image is lost, as shown in photos 2 and 3, it looks even worse than in the first post.

the emulator from this site is better than the version from https://xenia.jp/

I do not know what the difference is, I know one thing that finally the emulator stopped closing in the game
Naruto Rise of Ninja so you can test this title
in Naruto Broken Bond Some textures are apparently missing

while the emulator from https://xenia.jp/ is always closed to me in the game Naruto Rise Of Ninja
I don't know why, the files are the same, the emulator is the same, but these emulators are different.

just from this side the emulator is better, it does not close randomly.

but there are still bugs like in the pictures above, it is in every version.

**Comment by zerr10** [comment]
![obraz](https://user-images.githubusercontent.com/18278728/215618539-cfa080b2-9cf6-4fcf-9a94-c1bf4c1afca8.png)

![obraz](https://user-images.githubusercontent.com/18278728/215618588-5de5d814-73b2-4744-bfc3-b34a3d9203b9.png)

![obraz](https://user-images.githubusercontent.com/18278728/215618649-72533c4e-d702-4101-a404-ec734cd0944f.png)

last version emulator Build: canary_experimental@b10c84b34 no help same bug something can see but some is missing D :)

**Comment by zerr10** [comment]
![55](https://github.com/xenia-project/game-compatibility/assets/18278728/be9864ad-c589-43b3-a95a-e0a0c1ce5019)

I think it works now but I haven't tested the whole game...

congratulations for your work.

**Comment by Slackjaw1431** [comment]
Seems to be broken on all the newer releases.

**Comment by robertholden4** [comment]
Yeah my game does the same as above even when I try the older versions and crashes still happen but not as often. If this gets fixed would be great.

**Comment by robertholden4** [comment]
Alright guys not much talk on here on how to fix this but I found the answer on Reddit. To fix the issues above just press F5. Now I still crash a good amount so if anyone has a fix to that lmk, please.

**Comment by ndupded** [comment]
Naruto The Broken Bond is fully playable on canary_experimental@6e54153 November 7 2022 build(Xenia Canary). On the config change the d3d12_clear_memory_page_state from false to true to fix the graphical issues. The problem was their was dialogue cut out on the newer releases and for whatever reason this specific build does not have that dialogue problem. you may experience some freeze crashing here and there but I was able to do a full playthrough of the story mode on this build.

**Comment by bad361** [comment]
Tried both experimental builds in the comments but i can't seem to get pass this frame, the game freezes, tho the i can hear the gamplay sounds. Also vulkan works but has a lot of graphical issues.

![explorer_10_09_23_07_18](https://github.com/xenia-project/game-compatibility/assets/61082315/abf6413c-fcfd-4ccd-ae1b-a41f1c33543b)

**Comment by RandomAnimeGamer** [comment]
> Tried both experimental builds in the comments but i can't seem to get past this frame in the very beginning, the game freezes, though i can hear the gamplay sounds, but can't control. Also vulkan works but has a lot of graphical issues.
> 
> ![explorer_10_09_23_07_18](https://user-images.githubusercontent.com/61082315/273497495-abf6413c-fcfd-4ccd-ae1b-a41f1c33543b.png)

Did you try pausing and switching dub --> sub or sub --> dub? That fixes it for me.

**Comment by Willdkerler** [comment]
> Tried both experimental builds in the comments but i can't seem to get past this frame in the very beginning, the game freezes, though i can hear the gamplay sounds, but can't control. Also vulkan works but has a lot of graphical issues.
> 
> ![explorer_10_09_23_07_18](https://user-images.githubusercontent.com/61082315/273497495-abf6413c-fcfd-4ccd-ae1b-a41f1c33543b.png)

Hey, for me I just need to press F5 every time I get an issue, and it works from there.

**Comment by Vismayor** [comment]
With the current release of the Canary the cutscene bug is still there and the game randomly crashes. The F5 not working for me.

**Comment by Willdkerler** [comment]
> With the current release of the Canary the cutscene bug is still there and the game randomly crashes. The F5 not working for me.

Every time Audio Stops or game won't evolve I just change the voices from japanese to English or vice versa. 

and when graphical problems occur I just got F5.
Still works on current Canary build.

**Comment by Vismayor** [comment]
> > With the current release of the Canary the cutscene bug is still there and the game randomly crashes. The F5 not working for me.
> 
> Every time Audio Stops or game won't evolve I just change the voices from japanese to English or vice versa.
> 
> and when graphical problems occur I just got F5. Still works on current Canary build.

Not in my case. What about the random crashing? For me, it worked so long I left the village for the first time, and the game stopped.

**Comment by franc-drigue** [comment]
I solved the issue of graphics problems by setting this option below to TRUE:

[GPU]
clear_memory_page_state = true

**Comment by SoCalledFunnyman** [comment]
Can confirm that the most recent canary build (as of Sep 19 2024 9PM EST) seems to work when clear_memory_page_state = true, but freezing is frequent. Also, during cutscenes, you need to FREQUENTLY change between dub/sub. As in, "you're lucky if you get through a single line without needing to do it" frequent.

**Comment by Margen67** [comment]
Xenia Canary issue: https://github.com/xenia-canary/game-compatibility/issues/87

**Comment by Toemass202** [comment]
To remove the cutscene dialogue cut, set in the config 'use_new_decoder = true'.

**Comment by ts95** [comment]
> To remove the cutscene dialogue cut, set in the config 'use_new_decoder = true'.

Now it no longer freezes for me, nice. But the game still crashes sometimes in vs mode.

**Comment by bob-shink** [comment]
Hello, Am I the only one with a lagging game? 

But is sometimes not lagging

**Comment by PhantumCat** [comment]
Anyone knows why the hubworld music isn't playing?
It kinda restarts when I close the pause menu, but then it instantly stops again and I can't find a way to fix it

**Comment by hkmodd** [comment]
2025 and I can't still play the game at all with all those crashes and stuff T__T
